setup.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
```

Comment:
I don't think this is necessary?

Reply:
Strictly speaking, I suppose not. However, I put that so that it can be run like an executable on a Linux terminal. ./setup.py is less typing than python setup.py .